src/serverFilters/movieScreenings.js

```diff
@@ -0,0 +1,12 @@
+export default async function movieScreenings(payload, query) {
+  let data = payload.data;
+  const filteredArray = [];
+
+  data.forEach((i) => {
+    if (i.attributes.movie.data.id == query) {
+      filteredArray.push(i);
+    }
+  });
+
+  return filteredArray;
```

Comment:
This could all just be replaced with `return data.filter(i => i.attributes.movie.data.id == query)`.